typescript/ccip-server/README.md

```diff
@@ -0,0 +1,45 @@
+# CCIP-read service framework
+
+This package contains the service framework for the CCIP-read project, built off of the [CCIP-server framework](https://github.com/smartcontractkit/ccip-read). It allows building of any execution logic, given a Hyperlane Relayer call.
```

Comment:
A quick example or two or when this would be useful may be a nice addition here.

Reply:
Will add this in the another PR.